pkg/ui/src/util/events.ts

```diff
@@ -72,7 +72,10 @@ export function getEventDescription(e: Event$Properties): string {
     case eventTypes.NODE_RESTART:
       return `Node Rejoined: Node ${targetId} rejoined the cluster`;
     case eventTypes.SET_CLUSTER_SETTING:
-      return `Cluster Setting Changed: User ${info.User} set ${info.SettingName} to ${info.Value}`;
+      if (info.Value.length > 0) {
```

Comment:
info.Value might be null.  Either use lodash's `_.isEmpty()` function, or check `if (info.Value && info.Value.length > 0)`

Reply:
done